src/main/java/com/bandwidth/iris/sdk/IrisClient.java

```diff
@@ -1,186 +0,0 @@
-package com.bandwidth.iris.sdk;
-
-import com.bandwidth.iris.sdk.model.BaseModel;
-import com.bandwidth.iris.sdk.model.BaseResponse;
-import com.bandwidth.iris.sdk.utils.XmlUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.apache.http.Header;
-import org.apache.http.HttpResponse;
-import org.apache.http.auth.AuthScope;
-import org.apache.http.auth.Credentials;
-import org.apache.http.auth.UsernamePasswordCredentials;
-import org.apache.http.client.methods.*;
-import org.apache.http.client.utils.URIBuilder;
-import org.apache.http.entity.ContentType;
-import org.apache.http.entity.FileEntity;
-import org.apache.http.entity.StringEntity;
-import org.apache.http.impl.client.DefaultHttpClient;
-import org.apache.http.util.EntityUtils;
-
-import javax.xml.stream.XMLInputFactory;
-import java.io.File;
-import java.net.URISyntaxException;
-import java.util.HashMap;
-import java.util.Map;
-
-/**
- * Created by sbarstow on 9/22/14.
- */
-public class IrisClient {
-
-    private static String defaultUri = "https://api.inetwork.com/v1.0";
```

Comment:
And is there the equivalent test env url that I can verify changes against?

On Tue, Jul 7, 2015 at 10:49 AM, Frank Schmager <notifications@github.com>
wrote:

> In src/main/java/com/bandwidth/iris/sdk/IrisClient.java
> <https://github.com/bandwidthcom/java-bandwidth-iris/pull/1#discussion_r34048174>
> :
>
> > -import org.apache.http.entity.StringEntity;
> > -import org.apache.http.impl.client.DefaultHttpClient;
> > -import org.apache.http.util.EntityUtils;
> > -
> > -import javax.xml.stream.XMLInputFactory;
> > -import java.io.File;
> > -import java.net.URISyntaxException;
> > -import java.util.HashMap;
> > -import java.util.Map;
> > -
> > -/**
> > - * Created by sbarstow on 9/22/14.
> > - */
> > -public class IrisClient {
> > -
> > -    private static String defaultUri = "https://api.inetwork.com/v1.0";
>
> use new URL: https://dashboard.bandwidth.com/api
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bandwidthcom/java-bandwidth-iris/pull/1/files#r34048174>
> .
>



-- 
scottbarstow.com
@scottbarstow